packages/medusa/src/api/routes/admin/orders/get-order.ts

```diff
@@ -61,18 +63,479 @@ export default async (req, res) => {
+        "id",
+        "created_at",
+        "updated_at",
+        "rate",
+        "name",
+        "code",
+        "metadata",
+        "item_id",
+      ],
+    },
+    variant: expandedVariant,
+  }
+
+  const expandedShippingMethods = {
+    fields: ["id", "shipping_option_id", "price", "data"],
+    shipping_option: {
+      fields: [
+        "id",
+        "created_at",
+        "updated_at",
+        "deleted_at",
+        "name",
+        "region_id",
+        "profile_id",
+        "provider_id",
+        "price_type",
+        "amount",
+        "is_return",
+        "admin_only",
+        "data",
+        "metadata",
+      ],
+    },
+    tax_lines: {
+      fields: [
+        "id",
+        "created_at",
+        "updated_at",
+        "rate",
+        "name",
+        "code",
+        "metadata",
+        "shipping_method_id",
+      ],
+    },
+  }
+
+  const expandedShippingAddress = {
+    fields: [
+      "id",
+      "created_at",
+      "updated_at",
+      "deleted_at",
+      "customer_id",
+      "company",
+      "first_name",
+      "last_name",
+      "address_1",
+      "address_2",
+      "city",
+      "country_code",
+      "province",
+      "postal_code",
+      "phone",
+      "metadata",
+    ],
+  }
+
+  const expandedFullfilments = {
+    fields: [
+      "id",
+      "no_notification",
+      "provider_id",
+      "location_id",
+      "tracking_numbers",
+      "data",
+      "shipped_at",
+      "canceled_at",
+      "metadata",
+    ],
+    tracking_links: {
+      fields: ["id", "url", "tracking_number", "fulfillment_id", "metadata"],
+    },
+    items: {
+      fields: ["fulfillment_id", "item_id", "quantity"],
+      item: expandedItems,
+    },
+    provider: {
+      fields: ["id", "is_installed"],
+    },
+  }
+
+  const expandedGiftCard = {
+    fields: [
+      "code",
+      "value",
+      "balance",
+      "is_disabled",
+      "ends_at",
+      "tax_rate",
+      "metadata",
+    ],
+  }
+
+  const expandedPayment = {
+    fields: [
+      "id",
+      "created_at",
+      "updated_at",
+      "swap_id",
+      "cart_id",
+      "order_id",
+      "amount",
+      "currency_code",
+      "amount_refunded",
+      "provider_id",
+      "data,",
+      "captured_at",
+      "canceled_at",
+      "metadata",
+    ],
+  }
+
+  const expandedReturn = {
+    fields: [
+      "id",
+      "status",
+      "location_id",
+      "shipping_data",
+      "refund_amount",
+      "received_at",
+      "no_notification",
+      "metadata",
+    ],
+    items: {
+      fields: [
+        "return_id",
+        "item_id",
+        "quantity",
+        "is_requested",
+        "requested_quantity",
+        "received_quantity",
+        "reason_id",
+        "note",
+        "metadata",
+      ],
+      item: expandedItems,
+      reason: {
+        fields: ["id", "value", "label", "description", "metadata"],
+      },
+    },
+    shipping_method: expandedShippingMethods,
+  }
+
+  const query = {
```

Comment:
question: Are the current joiner config enough? All those corresponds to the relation we pass to the order service entry point right and in that case indeed the order config is just what we need am i right?